data/mods/Aftershock/items/gun/voltaic.json

```diff
@@ -24,5 +24,32 @@
     ],
     "ammo_effects": [ "ELECTRO_ANOMALY" ],
     "flags": [ "NEVER_JAMS", "NON_FOULING", "NEEDS_NO_LUBE" ]
+  },
+  {
+    "id": "afs_voltaic_pistol",
+    "type": "GUN",
+    "copy-from": "laser_rifle",
+    "name": { "str": "W3100 Volt" },
+    "description": "Wraitheon Armories premier electrolaser personal defense weapon.  The W3100 Volt adjustable power settings make it adaptable to most conflict situations, lethal or otherwise, and make it a favorite among well-off security forces and law enforcement corps.",
```

Comment:
```suggestion
    "description": "Wraitheon Armories' premier electrolaser personal defense weapon.  The W3100 Volt adjustable power settings make it adaptable to most conflict situations, lethal or otherwise, and make it a favorite among well-off security forces and law enforcement corps.",
```